### ISSUE TITLE
Error: Memo writing on image leads to error message 

### ISSUE THREAD

**Issue by jdittrich** [ccbogel/QualCoder#111]
**TL;DR:** Error message when OK or Canceling writing memos on _newly_ marked areas in image. 

**Version** branch master, 5th of July (I guess)

**Steps to reproduce:**
[assumes you have an image added via "manage Files" in main window]
1. Go to "Coding/Code Image"
2. Select an image
3. Select a code
4. Draw a new rectangle (the problem does not occur on "old" rectangles)
5. Right click, select memo
6. Write a memo (or not) click on _OK_ or _Cancel_

**Expected result:** Just saves silently, close memo window.
**Actual Result:** I get this error message (The app does not crash, though, I can continue) 

```
Traceback (most recent call last):
  File "/home/username/QualCoder-master/qualcoder/view_image.py", line 467, in eventFilter
    self.scene_context_menu(p)

  File "/home/username/QualCoder-master/qualcoder/view_image.py", line 493, in scene_context_menu
    self.coded_area_memo(item)

  File "/home/username/QualCoder-master/qualcoder/view_image.py", line 526, in coded_area_memo
    self.draw_coded_areas()

  File "/home/username/QualCoder-master/qualcoder/view_image.py", line 377, in draw_coded_areas
    rect_item.setPen(QtGui.QPen(color, 2, QtCore.Qt.DashLine))

TypeError: arguments did not match any overloaded call:
  QPen(): too many arguments
  QPen(Qt.PenStyle): argument 1 has unexpected type 'NoneType'
  QPen(Union[QBrush, QColor, Qt.GlobalColor, QGradient], float, style: Qt.PenStyle = Qt.SolidLine, cap: Qt.PenCapStyle = Qt.SquareCap, join: Qt.PenJoinStyle = Qt.BevelJoin): argument 1 has unexpected type 'NoneType'
  QPen(Union[QPen, QColor, Qt.GlobalColor, QGradient]): argument 1 has unexpected type 'NoneType'
  QPen(Any): too many arguments
```

**Comment by ccbogel** [comment]
Thanks Jan for identifying this.
Now fixed - it was trying to use an integer, but the integer was actually a string.
